phpunit/functional/Glpi/SocketTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
```

Comment:
Not sure this file should be under the `Glpi` directory (to reflect related class position in the tree).